cbrown/ex1/ex2.py

```diff
@@ -0,0 +1,9 @@
+#A comment, this is so you can read your programs later.
+#Anything after the # is ignored by python.
+
+print "I could have code like this." # and the comment after is ignored
+
+#You can also use a comment to "disable" or comment out a piece of code
+# print "This won't run."
+
+print "This will run."
```

Comment:
Notice that 🚫⏎ symbol? That means missing end of the line. It's proper to have an new line at the end of a file to prevent read errors in some situations